docs/dev-process/coding-guidelines.md

```diff
@@ -180,6 +180,21 @@ See [CODEOWNERS](../../.github/CODEOWNERS) for more information about the area e
   as it obscures the functional changes of the PR.
   A separate PR should be submitted for such refactoring without any functional changes.
 
+* Avoid initializing variables, fields and auto-implemented properties to zero or null because C# guarantees its are initialized to zero.
+
+* Follow Microsoft best practise guidances:
```

Comment:
Let's not say **follow** because some of those guidelines are contradicting with our coding convention. For example, the [naming guideline](https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/names-of-type-members) in the first link below says `"do not use "g_" or "s_" to indicate static fields"`, while we require static fields to start with `s_` or `t_`. 

I think it's better to say: `"Here are some useful links for your reference:"`

Reply:
Fixed.